scriptmodules/libretrocores/lr-kronos.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="lr-kronos"
+rp_module_desc="Saturn & ST-V emulator - Kronos port for libretro"
+rp_module_help="ROM Extensions: .iso .cue .zip .ccd .mds\n\nCopy your Sega Saturn & ST-V roms to $romdir/saturn\n\nCopy the required BIOS file saturn_bios.bin / stvbios.zip to $biosdir/kronos"
+rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro-mirrors/Kronos/extui-align/yabause/COPYING"
+rp_module_section="exp"
+rp_module_flags="!arm !aarch64"
+
+function sources_lr-kronos() {
+    gitPullOrClone "$md_build" https://github.com/libretro-mirrors/Kronos.git
```

Comment:
I noticed the default branch isn't master - please can you add the branch name to the command above as the retropie-setup script will assume master otherwise (which will cause issues with persistent repo option on retropie-setup).